### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by PfalconR3d** [Ryujinx/Ryujinx#7201]
### Overview

Ryujinx Portable Mode : Update and Rom Folders

### Smaller details

My suggestion is for a new folder for roms and updates were it would autodetect games and auto enable any updates in that folder .

### Nature of request

I was looking for a way to autodetect games that are in the portable folder without having to add the game directory , so that i could move the whole folder with the XCI files to another drive and not have to re-add the new path after moving .
I was told that the games folder is not for rom files and that there are no relative paths so what i want is not possible at this time .

### Why would this feature be useful?

Makes it very easy to have Portable installs of specific versions that work best for some games or are more stable .

**Comment by lostromb** [comment]
This seems very similar to https://github.com/Ryujinx/Ryujinx/pull/4099, can we merge the two efforts?

**Comment by PfalconR3d** [comment]
Sure , i just read through it , they look very similar in nature , i am a bit more specific for portable and just auto detecting games updates and dlc are present within a folder and auto enabling by default  , but what ever works .

That been said i have no idea how to merge the two issues together .

**Comment by J-Swift** [comment]
Yeah, I just volunteered to finish the linked issue and I'm not sure if it would fully address this or not. Maybe hold off until I wrap up on the other one before considering if this should be closed?

**Comment by lostromb** [comment]
Yeah, my ideal situation for managing game files + updates + dlcs is essentially:

1. You can have game, update, and DLC files in any combination of folders (i.e. they don't have to be in special "update" or "aoc" or "games" global directories). They would just be added as regular game/content directories using the existing config mechanisms.
2. Updates and DLCs are automatically detected and applied by default no matter which folder they are found in (though user can configure them after the fact)
3. To satisfy the portability requirements here, one possible design would be to allow `./` relative paths to be used as game folders - in which case they would resolve to the currently running Ryujinx working directory. And then in every other regard they would behave the same as above.

**Comment by PfalconR3d** [comment]
Yea being able to use ./ in the config.json  for game path would probably do at least half of what i am asking .

**Comment by J-Swift** [comment]
My current approach is in line with 1 and 2, so its likely 3 is what will
be unique here
